src/lwc/lwc_QuickAction/lwc_QuickAction.js

```diff
@@ -0,0 +1,28 @@
+
+import { LightningElement,api } from 'lwc';
+import { CloseActionScreenEvent } from 'lightning/actions';
+import createAccountShare from '@salesforce/apex/ShareAccountController.createAccountShare';
+
+export default class LwcQuickAction extends LightningElement {
+@api recordId;
+userOrGroupId;
+
+    handleChange(event){
+        this.userOrGroupId = event.target.value;
+    }
+
+    closeAction(){
+        this.dispatchEvent(new CloseActionScreenEvent());
+    }
+    Save(){
+        console.log(this.recordId);
```

Comment:
in real code all not necessary console.log or system.debug should be removed after development is done.